### DISCUSSION TITLE
希望能简化一下部署文档教程

### DISCUSSION THREAD

**Discussion by shulng** [dreamhunter2333/cloudflare_temp_email#495] [General]
文档的教程越看越懵
尤其是Github Actions,完全没看懂

**Comment by dreamhunter2333** [comment]
感谢反馈，三种方式选其中一种就可以，是哪些地方不清楚呢

**Reply by shulng** [reply]
> 感谢反馈，三种方式选其中一种就可以，是哪些地方不清楚呢

Github Actions部署方式没看懂
